integration/ksm/ksm_test.sh

```diff
@@ -0,0 +1,75 @@
+#!/bin/bash
+#
+# Copyright (c) 2019 Intel Corporation
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+set -e
+
+dir_path=$(dirname "$0")
+source "${dir_path}/../../lib/common.bash"
+source "${dir_path}/../../metrics/lib/common.bash"
+
+TEST_NAME="Run containers with KSM"
+KATA_KSM_THROTTLER="${KATA_KSM_THROTTLER:-yes}"
+RUNTIME="${RUNTIME:-kata-runtime}"
+PAYLOAD_ARGS="${PAYLOAD_ARGS:-tail -f /dev/null}"
+IMAGE="${IMAGE:-busybox}"
+WAIT_TIME="60"
+
+function setup() {
+	clean_env
+
+	check_processes
+
+	save_ksm_settings
+
+	set_ksm_aggressive
+
+	check_for_ksm
+
```

Comment:
nit: extra blank line. In fact, you could just compress all these lines in this function without blanks...